config/hierarchicalModel.js

```diff
@@ -0,0 +1,62 @@
+import { NAMESPACES as ns } from '../env';
+
+export default [
+  {
+    type: ns.ere`EredienstMandataris`,
+    children: [
+      {
+        path: ns.mandaat`isBestuurlijkeAliasVan`,
```

Comment:
This is interesting; and comes close concept-scheme export in the maintainer.
https://github.com/lblod/delta-producer-publication-graph-maintainer?tab=readme-ov-file#why-must-the-generated-deltas-of-the-application-stack-be-rewritten-by-the-producer
(Not saying that the mentioned solution is better; but the problem is recurring one, might see what we can extract from that)

Reply:
@bdevloed FYI; there are similarities with the transform in the delta-consumer too.

Reply:
Very close apparently. Didn't know about the concept scheme filtering on the producer.